<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefEffectsDlg</class>
 <widget class="QWidget" name="DlgPrefEffectsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1434</width>
    <height>823</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Effects Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="effectsTabs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="chainPresetsTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Effect Chain Presets</string>
      </attribute>
      <layout class="QGridLayout" name="chainPresetGridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="chainPresetsTabHeader">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Drag and drop to rearrange lists and copy chains between lists. Create and edit chain presets in the effect units in the main window. Please refer the manual for further details.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="presetListControllerLabel">
         <property name="text">
          <string>Chain presets from these lists will be selectable in the given order in the main window and from controllers (depending on the controller mapping).</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="chainPresetsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Effect Chain Presets</string>
         </property>
         <property name="topMargin" stdset="0">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QListView" name="quickEffectListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <layout class="QVBoxLayout" name="chainDetailsVBoxLayout">
         <item>
          <widget class="QLabel" name="effectsInChainHeader">
           <property name="text">
            <string>Effects in this chain preset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="effect1Name">
           <property name="text">
            <string>effect 1 name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="effect2Name">
           <property name="text">
            <string>effect 2 name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="effect3Name">
           <property name="text">
            <string>effect 3 name</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="chainButtonsTopRow">
           <item>
            <widget class="QPushButton" name="chainPresetImportButton">
             <property name="text">
              <string>Import</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupPresetActions</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chainPresetRenameButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Rename</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupPresetActions</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="chainButtonsBottomRow">
           <item>
            <widget class="QPushButton" name="chainPresetExportButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupPresetActions</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chainPresetDeleteButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupPresetActions</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="quickEffectChainPresetsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Quick Effect Chain Presets</string>
         </property>
         <property name="topMargin" stdset="0">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QListView" name="chainListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="visibleEffectsTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Visible Effects</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="effectsTablesGridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="effectsTablesHeaderLabel">
           <property name="text">
            <string>Drag and drop to rearrange lists and show or hide effects.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="visibleEffectsTableLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Visible Effects</string>
           </property>
           <property name="topMargin" stdset="0">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="hiddenEffectsTableLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Hidden Effects</string>
           </property>
           <property name="topMargin" stdset="0">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QTableView" name="visibleEffectsTableView"/>
         </item>
         <item row="2" column="1">
          <widget class="QTableView" name="hiddenEffectsTableView"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Effect Info</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="effectNameLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Name:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="effectName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string notr="true">(effect name)</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="effectVersionLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Version:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="effectVersion">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string notr="true">(effect version)</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="effectTypeLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Type:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="effectType">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string notr="true">(effect type)</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="effectAuthorLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Author:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="effectAuthor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string notr="true">(effect author)</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="effectDescriptionLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Description:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="effectDescription">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string notr="true">(effect description)</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupEffectKnobOptions">
     <property name="title">
      <string>Effect load behavior</string>
     </property>
     <layout class="QGridLayout" name="verticalLayoutEffectKnobOptions">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonKeepSuperknobPosition">
        <property name="text">
         <string>Keep superknob position</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupSuperKnob</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonSuperknobLoadDefault">
        <property name="text">
         <string>Reset superknob to preset default</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupSuperKnob</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonKeepMetaknobPosition">
        <property name="text">
         <string>Keep metaknob position</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupMetaKnob</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButtonMetaknobLoadDefault">
        <property name="text">
         <string>Reset metaknob to effect default</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupMetaKnob</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chainListView</tabstop>
  <tabstop>quickEffectListView</tabstop>
  <tabstop>visibleEffectsTableView</tabstop>
  <tabstop>hiddenEffectsTableView</tabstop>
  <tabstop>chainPresetImportButton</tabstop>
  <tabstop>chainPresetExportButton</tabstop>
  <tabstop>chainPresetRenameButton</tabstop>
  <tabstop>chainPresetDeleteButton</tabstop>
  <tabstop>radioButtonKeepSuperknobPosition</tabstop>
  <tabstop>radioButtonSuperknobLoadDefault</tabstop>
  <tabstop>radioButtonKeepMetaknobPosition</tabstop>
  <tabstop>radioButtonMetaknobLoadDefault</tabstop>
 </tabstops>
 <resources/>
 <buttongroups>
  <buttongroup name="buttonGroupSuperKnob"/>
  <buttongroup name="buttonGroupMetaKnob"/>
  <buttongroup name="buttonGroupPresetActions"/>
 </buttongroups>
 <connections/>
</ui>
